src/main/java/com/digital/home/controller/DailyTransactionController.java

```diff
@@ -18,4 +16,10 @@ public DailyTransaction getDailyTransaction(@PathVariable(value = "projectId") L
         return dailyTransactionService.getDailyTransactionDetails(projectId);
     }
 
+    @PostMapping("/daily-transactions/{projectId}")
```

Comment:
Add date as additional parameter

Reply:
As per BRD, date is non-editable and so system date is taken in the back end.